mpmp/src/cmds/SellPlot.java

```diff
@@ -0,0 +1,86 @@
+package cmds;
+
+import java.util.Arrays;
+import main.Client;
+import main.Conn;
+import main.ErrCode;
+import model.Player;
+import model.Plot;
+import view.Displayer;
+
+/**
+ * C->S
+ * @author Leander
+ */
+public class SellPlot implements CmdFunc{
+	private Displayer d;
+
+	@Override
+	public void exec(String line, Conn conn) {
+		String[] args = line.split(" ");
+		int price = -1;
+		
+		Player p = Player.search(((Client) conn).getName());
+		if (!p.isPlayer()) {
+			conn.sendErr(ErrCode.NotAPlayer);
+			return;
+		}
+		
+		if (args.length < 4) {
+			conn.sendErr(ErrCode.Usage, "sell-plot <Name des Grundstückes> <Käufer> <Preis>");
+			return;
+		}
+		
+		String plotname = Plot.matches(String.join(" ", Arrays.copyOfRange(args, 1, args.length)));
+		if (plotname == null) {
+			conn.sendErr(ErrCode.NotAPlot);
+			return;
+		}
+		
+		Plot plot = Plot.search(plotname);
+		
+		if (!p.equals(plot.getOwner())) {
+			conn.sendErr(ErrCode.AlreadyOwned, plot.getOwner().getName());
+			return;
+		}
+		
+		if (plot.getHouses() != 0) {
+			conn.sendErr(ErrCode.PlotWithHouse);
+			return;
+		}
+		
+		String[] s1 = line.split(plotname + " ");
+		String buyername = Player.matches(s1[1]);
```

Comment:
`Player.matches` has the embedded assumption that player names start with an `@`.